django_saml2_auth/views.py

```diff
@@ -96,7 +96,7 @@ def _get_metadata():
 
 # BEGIN TESORIO CHANGES
 # def _get_saml_client(domain):
-def _get_saml_client(domain, metadata_conf_url):
+def _get_saml_client(domain, metadata_conf_url, metadata_conf_raw):
```

Comment:
Since this is now a required param - will this break existing applications? I worry about backwards compatibility with this change here

Reply:
Good call, addressed, now it has None as the default value.